### DISCUSSION TITLE
大佬，按照你的说明自己尝试编译了一下

### DISCUSSION THREAD

**Discussion by shichaoufo** [ywmoyue/biliuwp-lite#346] [feature]
### 问题描述

发现没法编译，会报错，请问是少安装了什么软件吗
![image](https://github.com/ywmoyue/biliuwp-lite/assets/18585420/30c54b40-79b5-4090-9eb7-20ccd91b2124)

### 描述解决方案

希望能正常编译

### 代码贡献

否

**Comment by GD-Slime** [comment]
缺少了SDK. 需要在Visual Studio Installer里勾选并安装相关选项。
我目前勾选的有这几个，可以正常编译，仅供参考：
![image](https://github.com/ywmoyue/biliuwp-lite/assets/82302542/cd8f0522-4001-4c8b-8b7d-832f0389781f)

**Reply by shichaoufo** [reply]
大佬，我按照图片里的安装了，但是问题依旧，难道是需要配置环境变量什么的

**Reply by ywmoyue** [reply]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/4ee6759f-d0e9-4f1c-8875-0d25f4d601f6)

@shichaoufo  这是我这边的，你试试

**Reply by shichaoufo** [reply]
大佬，还是不行，然后我发现是代码里这行提示有问题，貌似是找不到Windows.Storage，如果我输入Windows后再输入点，弹出的框里面是找不到Storage的，这是缺了什么啊
![image](https://github.com/ywmoyue/biliuwp-lite/assets/18585420/3d9d2782-c25c-4c90-9327-713a98399f1f)

**Reply by ywmoyue** [reply]
@shichaoufo 试试安装这个https://go.microsoft.com/fwlink/p/?LinkID=2033908

**Reply by shichaoufo** [reply]
还是不行，又发现引用里面这个带感叹号
![image](https://github.com/ywmoyue/biliuwp-lite/assets/18585420/8aa32255-66b4-4a18-a1c9-4ec2a83ed3dd)

**Comment by ywmoyue** [comment]
文档问题，转discussion

**Reply by ywmoyue** [reply]
后续我调整下文档吧
